resource.html

```diff
@@ -0,0 +1,1051 @@
+                                <img src="./assets/latest-post4.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="">
+                            </a>
+                            </div>
+                        </div>
+                        <div class="cmp-teaser__content">
+                            <h2 class="cmp-teaser__title">
+                            <a class="cmp-teaser__title-link" href="/content/leaderimpact/lang/en2/resources/leading-through-a-crisis/video-series.html">Leading in Times of Crisis Video Series</a>
+                            </h2>
+                            <div class="cmp-teaser__description"><p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna.</p>
+                            </div>
+                        </div>
+                        </div>
+                    </div>
+                    </div>
+                </div>
+                </div>
+            </div>
+
+            <!-- Alternate ARTICLE SLIDER option -->
+            <div class="button bottom-btn aem-GridColumn aem-GridColumn--default--12 layout-grid" style="display: none;">
+                <a id="button-e7ea1382b31" class="cmp-button" href="/content/leaderimpact/lang/en2/resources.html">
+                <span class="cmp-button__text">Shop All</span>
+                </a>
+            </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="container responsivegrid featured-products aem-GridColumn aem-GridColumn--default--12">
+        <div class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+                <div class="responsivegrid aem-GridColumn product-title aem-GridColumn--default--12 layout-grid">
+                    <div class="cmp-container">
+                        <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="title-bcb754f8400" class="cmp-title">
+                                <h5 class="cmp-title__text">Featured Products</h5>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class=" responsivegrid product-slider-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div class="cmp-container">
+                        <div class="container responsivegrid product-item aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div class="cmp-container">
+                                <div class="teaser aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div class="cmp-teaser">
+                                        <div class="cmp-teaser__image post-product__image">
```

Comment:
@Prystal254 The image/content order needs correcting and the anchor needs to be in the right place. And `.post-product__image` can't go here. Custom classes can only go on the root of the component.